en/coding-the-legislation/writing_yaml_tests.md

```diff
@@ -156,23 +156,16 @@ Values can be arithmetic expressions too.
 
 ## Running a test
 
-The script `test_yaml.py` can run a single YAML test file, or all.
+To run yaml tests, use the command line tool `openfisca-run-test`, documented [here](http://openfisca.readthedocs.io/en/latest/openfisca-run-test.html) :
```

Comment:
I'm not sure “yaml tests” is very clear. What are the other types of tests? Do we really need to specify “yaml tests”?

In any case, the proper casing is `YAML` (uppercase).